tests/unit/harness_freebsd_ports

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh -e
```

Comment:
This wrapper exists because I need to override shell built-ins, but I can't source the script under test from the test file (it has to be a subprocess), and you cannot export functions in Bourne. It's a bit weird. Let me know if I need to make any parts clearer.